source/api/extension/text-granularity/index.md

```diff
@@ -0,0 +1,158 @@
+{:toc}
+
+## 1. Introduction
+
+### 1.1 Objectives and Scope
+
+One use of annotation in IIIF is to associate text with IIIF resources, with the text originating from optical character recognition (OCR) software, manual transcription, existing digitized text, or other sources. Furthermore, it is a common practice to produce sets of annotations at a particular level of ‘text granularity’; that is to say that each annotation in the set references the same unit of text, such as a character, line, or page. This extension recommends a pattern for indicating the level of text granularity of an annotation.
+
+### 1.2 Motivating Use Cases
+
+A number of common workflows can result in annotation sets with discrete levels of text granularity:
+
+- OCR text is frequently available at a range of granularities, from page to character.
+- Manual transcription user interfaces may constrain user input to the level of the line,
+block, or page.
+- Transcriptions may be produced without coordinate data and thus have a very coarse level of granularity, such as page- or block-level.
+
+Identification of the level of text granularity in published annotations can facilitate the reuse of their textual content and target regions in other applications. Search is a primary use case, with the word-level annotations allowing for the accurate highlighting of search result terms in the user interface. Software designed for user interaction and input, such as an application designed to allow users to correct OCR text, might suppress finer-grained levels and make use of paragraph- or block-level text to simplify editing.
+
+## 2. Text Granularity Levels and the `textGranularity` Property
+
+The `textGranularity` property identifies the Text Granularity Level of a resource. The value _MUST_ be a single string. This extension defines the Text Granularity Levels found in the table below. The string _SHOULD_ be one of those defined in the table below or in the extensions registry.
+
+| Text Granularity Level       |  Description   |
+|------------------------------|-----------------
+| `page`                       |  A page in a paginated document |
+| `block`                      |  An arbitrary region of text    |
+| `paragraph`                  |  A paragraph                    |
+| `line`                       |  A topographic line             |
+| `word`                       |  A single word                  |
+| `glyph`                      |  A single glyph or symbol       |
+{: .api-table #table-granularity-level-dfn}
+
+```json-doc
+{ "textGranularity": "line" }
+```
+
+## 3. Use of the `textGranularity` Property with Annotations
+
+An Annotation _MAY_ have the `textGranularity` property. An Annotation that has the property _SHOULD_ target an IIIF Presentation API Canvas or segment and the identified Text Granularity Level _SHOULD_ describe that of the textual content represented by the content resources painted on the Target.
```

Comment:
"target" -- is that a word we use? "reference"? "be associated with"?

Reply:
Changing to `..._SHOULD_ reference a IIIF Presentation API Canvas or segment in the `target` property and...`